docs/integrations.md

```diff
@@ -210,4 +212,5 @@ flytesnacks/examples/snowflake_plugin/index
 flytesnacks/examples/databricks_plugin/index
 flytesnacks/examples/bigquery_plugin/index
 flytesnacks/examples/airflow_plugin/index
+flytesnacks/examples/flyin_plugin/index
```

Comment:
@cosmicBboy The workflow for adding a new plugin example is a bit involved. We have to update `flyte's` TOC with an example that does not exist and then add the example to `flytesnacks`.

Can this TOC be auto generated from `flytesnacks` during the doc build process? (Implementation-wise it would auto generate a TOC file that we `.. include::` in this spot.)